pkgs/by-name/ko/koboldcpp/package.nix

```
@@ -0,0 +1,98 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, autoPatchelfHook
+, makeWrapper
+, python311Packages
+, tk
+
+, config
+, cudaPackages ? { }
+
+, openblasSupport ? true
+, openblas
+
+, cublasSupport ? config.cudaSupport
+
+, clblastSupport ? true
```

```suggestion
, clblastSupport ? stdenv.isLinux
```

`clblast` is only available on `platforms.linux` https://github.com/NixOS/nixpkgs/blob/5710852ba686cc1fd0d3b8e22b3117d43ba374c2/pkgs/development/libraries/science/math/clblast/default.nix#L42